press/agent.py

```diff
@@ -101,6 +103,21 @@ def reinstall_site(self, site):
 			site=site.name,
 		)
 
+	def check_database_file(self, site):
+		remote_database_file = site.remote_database_file
+		mem_database_file = requests.get(url = frappe.get_doc("Remote File", site.remote_database_file).download_link)
```

Comment:
umm, this could be in GBs :sweat_smile: 

Maybe validate this on server before restore?

Reply:
Yep. Will have to rewrite in agent. Should've marked this as  a draft PR :P